### PR TITLE
Added `options.refresh` parameter for `popPage()`

### PR DESCRIPTION
**Pull request by frankdiox**

### CODE REVIEW

File: framework/directives/navigator.js [hunk 1]
```diff
@@ -260,6 +260,9 @@
  * @param {String} [options.animationOptions]
  *   [en]Specify the animation's duration, delay and timing. E.g.  <code>{duration: 0.2, delay: 0.4, timing: 'ease-in'}</code>[/en]
  *   [ja]アニメーション時のduration, delay, timingを指定します。e.g. <code>{duration: 0.2, delay: 0.4, timing: 'ease-in'}</code> [/ja]
+ * @param {Boolean} [options.refresh]
+ *   [en]The previous page will be refreshed (destroyed and created again) before popPage action.[/en]
+ *   [ja][/ja]
```
Comment (frankdiox) [inline framework/directives/navigator.js]: Missing Japanese translation.

**Comment by argelius** [comment]
@frankdiox Is this done?

**Comment by frankdiox** [comment]
@argelius Yes, I think so!

**Comment by argelius** [comment]
Thanks @frankdiox !

**Pull request merged**